pylti/flask.py

```diff
@@ -68,7 +68,7 @@ def name(self):
             return ''
 
     @property
-    def user_id(self):
+    def user_id(self):  # pylint: disable=no-self-use
```

Comment:
for the `@property` methods, I think you are stuck using self anyway, so these disables are ok